tasks/eva_3085/eva3085/pom.xml

```diff
@@ -8,6 +8,12 @@
     <packaging>jar</packaging>
 
     <dependencies>
+        <dependency>
+            <groupId>uk.ac.ebi.ampt2d</groupId>
+            <artifactId>accession-commons-mongodb</artifactId>
+            <version>0.7.9-SNAPSHOT</version>
```

Comment:
This should be pulled in as a transitive dependency from eva-accession-deprecate anyway. Also, I can't seem to find the snapshot version in the [accession-commons repo](https://github.com/EBIvariation/accession-commons/blob/master/accession-commons-mongodb/pom.xml)